lib/recurly.js

```diff
@@ -335,7 +335,9 @@ export class Recurly extends Emitter {
       return;
     }
 
-    if (!this.fraud) this.fraud = new Fraud(this);
+    if (!this.fraud || !this.fraud.dataCollectorInitiated) {
```

Comment:
If fraud options gets passed when recurly.js is first initialized, the recurly object contains a fraud object, but nothing has been attached and dataCollector hasn't been initialized. If `recurly.configure` gets called after it's already been called, this check will always return false, and fraud will never reconfigure and attach.